pkg/controller/v1beta1/inferenceservice/utils/utils.go

```diff
@@ -77,9 +77,7 @@ func GetDeploymentMode(annotations map[string]string, deployConfig *v1beta1api.D
 func MergeRuntimeContainers(runtimeContainer *v1.Container, predictorContainer *v1.Container) (*v1.Container, error) {
 	// Default container configuration from the runtime.
 	coreContainer := v1.Container{
```

Comment:
Not necessarily part of this PR, but just now noticing that since the runtimeContainer type is now v1.Container (instead of the subset of fields via v1alpha1.Container), if a user uses a field not listed here in a ServingRuntime, it would be dropped.

Perhaps can just update here to something like `coreContainer := runtimeContainer.DeepCopy()` or  maybe even use `runtimeContainer` directly?

Reply:
I updated this in the latest commit.